config/ext.json

```diff
@@ -446,18 +446,11 @@
         ]
     },
     "opentelemetry": {
-        "notes": true,
         "support": {
-            "Windows": "wip"
+            "BSD": "wip"
         },
-        "source": "opentelemetry",
-        "unix-only": true,
-        "ext-suggests": [
```

Comment:
This should be kept as its part of the optional extension dependencies:

![image](https://github.com/user-attachments/assets/21354f43-4439-494e-9117-a91b32a79eae)

Reply:
I didn't find any part in the source code that depends on the other extensions above. Maybe the document says that there are optional acceleration combinations, but this does not belong to "optional extensions at the build level"